sheerwater_benchmarking/data/stage_iv.py

```diff
@@ -0,0 +1,123 @@
+"""Stage IV precipitation data product."""
+import numpy as np
+import xarray as xr
+from dateutil import parser
+
+from sheerwater_benchmarking.utils import apply_mask, cacheable, clip_region, dask_remote, regrid, roll_and_agg
+
+
+@dask_remote
+@cacheable(data_type='array',
+           cache_args=['year', 'grid'],
+           chunking={'lat': 300, 'lon': 300, 'time': 365})
+def stage_iv_gridded(year, grid):
+    """Regridded Stage IV precipitation data by year.
+
+    Reads data from USGS OSN zarr store.
+    Data is available from 2002-present.
+    """
+    # Open the zarr store
+    store = 'https://usgs.osn.mghpcc.org/mdmf/gdp/stageiv_combined.zarr/'
```

Comment:
No need for ending slash

Reply:
Maybe we could make a practice of leaving a comment with some few details about the data store here

Reply:
I removed the slash and updated the docstring with some more details about where the data came from.